### ISSUE TITLE
Configure MAMP for local development

### ISSUE THREAD

**Issue by decabeza** [mhoofman/wordpress-heroku#16]
Hi, 

I follow all the steps and it's ok. Wordpress is installed on heroku and works correctly.

The problem is that I don't know how to configure MAMP for development in my local machine and then push to heroku.
- When i go to: http://localhost:8888/myapp/ appears:

Connecting to your PostgreSQL database without a password is considered insecure. 
If you want to do it anyway, please set "PG4WP_INSECURE" to true in your "db.php" file.
- If I change PG4WP_INSECURE for true appears the follow error:

Error establishing a database connection
- And in: http://localhost:8888/myapp/wp-admin/ appears:

Can’t select database

We were able to connect to the database server (which means your username and password is okay) but not able to select the database.

Are you sure it exists?
Does the user have permission to use the database?
On some systems the name of your database is prefixed with your username, so it would be like username_. Could that be the problem?
If you don't know how to set up a database you should contact your host. If all else fails you may find help at the WordPress Support Forums.
- How can i configure it?

I run $ heroku congif

DATABASE_URL        => [database type]://[username]:[password]@[host]:[port]/[database name]
SHARED_DATABASE_URL => [database type]://[username]:[password]@[host]:[port]/[database name]

Ok, but... how put this data in "wp-config.php"??

Thank you very much!

Regards.

Alberto.

**Comment by kintso** [comment]
Hi @SteveArnold5,

The URL you provided points to an error. Could you please share how you did it?

**Comment by jiang925** [comment]
I run my heroku-wordpress locally on a Raspberry Pi. I inject environment variables using PHP's www.conf file. I see people complaints about having this issue on Mac as well. The solution should be the same. Replace the $_ENV on the $db line with getenv.

I wrote about the whole process, with environment variable injection as well in here: http://blog.jiang925.com/2015/06/01/run-heroku-version-of-wordpress-locally-on-raspberry-pi-fixed-cant-select-database-problem/.

I forked this project to get it to work: https://github.com/jiang925/wordpress-heroku. Once I figure out how to get pull request to work, I'll submit a pull request.
